packages/plugin-conversation/src/process-image.browser.js

```diff
@@ -81,8 +82,12 @@ export default function processImage({file, thumbnailMaxWidth, thumbnailMaxHeigh
       canvas.height = thumbnailDimensions.height;
 
       const ctx = canvas.getContext(`2d`);
-      ctx.drawImage(img, 0, 0, thumbnailDimensions.width, thumbnailDimensions.height);
-
+      if (file && file.image && file.image.orientation && file.image.orientation !== 1) {
```

Comment:
does it make sense to hide this if-block in helper-image and just make setImageOrientation a noop if file.image.orientation === 1?